src/build/makers/firefoxos.js

```diff
@@ -0,0 +1,34 @@
+var shell        = require('shelljs'),
+    path         = require('path'),
+    error_writer = require('./error_writer'),
+    n            = require('ncallbacks'),
+    fs           = require('fs'),
+    mspec        = require('./mobile_spec');
```

Comment:
This guy is a directory, so the script fails with `Error: Cannot find module './mobile_spec'`.